misc/userscripts/dmenu_qutebrowser

```diff
@@ -38,9 +38,10 @@
 #				(This is unnecessarily long. I use this rarely, feel free to make this script accept parameters.)
 #
 
-[ -z "$QUTE_URL" ] && QUTE_URL='http://google.com'
 
-url=$(echo "$QUTE_URL" | cat - "$QUTE_CONFIG_DIR/quickmarks" "$QUTE_DATA_DIR/history" | dmenu -l 15 -p qutebrowser)
+[ -z "$QUTE_URL" ] && QUTE_URL='https://duckduckgo.com'
+
+url=$(printf "%s\n%s" "$QUTE_URL" "$(sqlite3 "$QUTE_DATA_DIR/history.sqlite" 'select url from CompletionHistory')" | cat "$QUTE_CONFIG_DIR/quickmarks" - | dmenu -l 15 -p qutebrowser)
```

Comment:
```suggestion
url=$(printf "%s\n%s" "$QUTE_URL" "$(sqlite3 -separator ' ' "$QUTE_DATA_DIR/history.sqlite" 'select title, url from CompletionHistory')" | cat "$QUTE_CONFIG_DIR/quickmarks" - | dmenu -l 15 -p qutebrowser)
```

Reply:
As you also wrote in #3091, do you think that both qutedmenu and dmenu_qutebrowser is required? @The-Compiler

Reply:
I don't really have an opinion on that - given that it took ~3 years for someone to update them after the history format change, I doubt they're in much use :smile: 

Seems fine to me to delete the other one instead, as they do seem to be pretty similar - any particular reason why you went for this rather than the other one?

Reply:
I had the longest name, and I'm drawn to long things if you know what I mean :smirk:
jokes aside, I don't really have a specific reason why I chose that one above any other. I'm pretty new to qb, and wanted to implement a keybind in my i3 config to access the dmenu tool without having to have an instance of qb.

And as I was modifying the script I found that it didn't access the history :man_shrugging:

Reply:
Finally got around to fix this,  see #aacc68a :)

Reply:
@The-Compiler I know the issue that this fixes is low priority, but would you mind taking another look at this? :smile:

Reply:
I will once I get back to PRs - sometimes I happen to work on other stuff (new features, other maintenance stuff, things outside qutebrowser, weekends not spent before a screen, etc.). Requesting a review is enough, no need to highlight me. :wink:

Reply:
Yes, of course. my bad. I didn't really know if it was enough to request that you re-review it. Whenever you have time :slightly_smiling_face: